ppocr/utils/loggers/wandb_logger.py

```diff
@@ -40,12 +44,14 @@ def __init__(self,
 
         if self.config:
             self.run.config.update(self.config)
+        
+        self.log_checkpoint = log_checkpoint
 
     @property
     def run(self):
         if self._run is None:
             if self.wandb.run is not None:
-                logger.info(
+                logging.info(
```

Comment:
It is recommended to use the inner logger to print, which can be initialized with the following code
```python
from ppocr.utils.logging import get_logger
logger = get_logger()
```